crates/aggchain-proof-contracts/src/lib.rs

```diff
@@ -274,18 +279,14 @@ where
 
 async fn host_execute<C: SolCall, P: Provider<AnyNetwork> + Clone>(
     contract_address: Address,
-    host_executor: &mut HostExecutor<P>,
+    sketch: &mut EvmSketch<P>,
```

Comment:
This no longer needs to be `mut`.
```suggestion
    sketch: &EvmSketch<P>,
```